buildpack/centos8/Dockerfile

```diff
@@ -52,9 +52,7 @@ RUN yum -y install \
 
 # Python and tools
 RUN yum -y install python3 python3-devel python3-wheel && \
-    wget -qO - https://bootstrap.pypa.io/get-pip.py | python3 && \
-    pip install --upgrade "pip==19.1.1" && \
-    pip install setuptools virtualenv && \
+    pip3 install virtualenv && \
```

Comment:
Per https://github.com/StackStorm/st2packaging-dockerfiles/blob/78ed652b4a4c575a36b3ce2ec4be2ee93772cfcb/packagingbuild/centos8/Dockerfile#L90-L92
in community build dockerfiles

Can we also pin pip and virtualenv to have the same consistent environment across the builds?

Reply:
`pip==19.1.1` causes make rpm build error so I will not include that.  `rpmdevtools` is already up on the build tools list in the same Dockerfile.